### PR TITLE
Pandaria: Update go mod

### PR DESCRIPTION
**Pull request by smallteeths**
## Issue: 
https://github.com/cnrancher/pandaria-images-scanning/issues/43
 
## Problem
| image                       | rancher_release     | package_name                    | type                   | vulnerability_id | severity | url                                           | target           | patched_version                        | mirrored | state           | notes |
|-----------------------------|---------------------|---------------------------------|------------------------|------------------|----------|-----------------------------------------------|-----------------|----------------------------------------|----------|----------------|-------|
| cnrancher/ack-operator:v2.7.2-ent-rc1 |                     | github.com/emicklei/go-restful | suse linux enterprise server | CVE-2022-1996    | CRITICAL | https://avd.aquasec.com/nvd/cve-2022-1996     | usr/bin/ack-operator | 2.16.0                                 | False    | pending review |       |
| cnrancher/ack-operator:v2.7.2-ent-rc1 |                     | github.com/rancher/wrangler    | suse linux enterprise server | CVE-2022-31249   | HIGH     | https://avd.aquasec.com/nvd/cve-2022-31249    | usr/bin/ack-operator | "0.7.4-security1, 0.8.5-security1, 0.8.11, 1.0.1" | False    | pending review |       |
| cnrancher/ack-operator:v2.7.2-ent-rc1 |                     | golang.org/x/net               | suse linux enterprise server | CVE-2022-27664   | HIGH     | https://avd.aquasec.com/nvd/cve-2022-27664    | usr/bin/ack-operator | 0.0.0-20220906165146-f3363e06e74c       | False    | pending review |       |
| cnrancher/ack-operator:v2.7.2-ent-rc1 |                     | golang.org/x/text              | suse linux enterprise server | CVE-2022-32149   | HIGH     | https://avd.aquasec.com/nvd/cve-2022-32149    | usr/bin/ack-operator | 0.3.8                                  | False    | pending review |       |
| cnrancher/ack-operator:v2.7.2-ent-rc1 |                     | gopkg.in/yaml.v3                | suse linux enterprise server | CVE-2022-28948   | HIGH     | https://avd.aquasec.com/nvd/cve-2022-28948    | usr/bin/ack-operator | 3.0.0-20220521103104-8f96da9f5d5e       | False    | pending review |       |

### Manual Testing
python ./run.py -c output.md -i ./image.txt -d -r docker.io -t xx

### CODE REVIEW

**Comment by smallteeths** [comment]
自测结果如下

```
trivy image --no-progress --ignore-unfixed --severity HIGH,CRITICAL --security-checks vuln --exit-code 1 wangsiye/ack-operator:v2.7.0-update.50
2023-03-07T08:42:47.650Z	WARN	'--security-checks' is deprecated. Use '--scanners' instead.
2023-03-07T08:42:47.658Z	INFO	Vulnerability scanning is enabled
2023-03-07T08:42:49.547Z	INFO	Detected OS: suse linux enterprise server
2023-03-07T08:42:49.547Z	INFO	Detecting SUSE vulnerabilities...
2023-03-07T08:42:49.548Z	INFO	Number of language-specific files: 1
2023-03-07T08:42:49.548Z	INFO	Detecting gobinary vulnerabilities...

wangsiye/ack-operator:v2.7.0-update.50 (suse linux enterprise server 15.4)

Total: 0 (HIGH: 0, CRITICAL: 0)
```

File: go.mod [hunk 1]
```diff
@@ -2,20 +2,20 @@ module github.com/cnrancher/ack-operator
...
 
 require (
 	github.com/alibabacloud-go/cs-20151215/v3 v3.0.31
 	github.com/alibabacloud-go/tea v1.1.20
 	github.com/aliyun/alibaba-cloud-sdk-go v1.62.159
 	github.com/google/uuid v1.3.0
-	github.com/rancher/lasso v0.0.0-20220627205005-00d9c8e9dda6
-	github.com/rancher/wrangler v1.0.1-0.20220520195731-8eeded9bae2a
+	github.com/rancher/lasso v0.0.0-20221227210133-6ea88ca2fbcc
+	github.com/rancher/wrangler v1.0.2
```
Comment (Jason-ZW) [inline go.mod]: 这里最好可以跟cnrancher/pandaria 主线保持一致

**Review by Jason-ZW — CHANGES_REQUESTED** [review]
Others LGTM

File: go.mod (hunk 1 above)
Comment (smallteeths) [reply go.mod]: github.com/rancher/wrangler v1.0.2 

我看了 gke 也用了最新的版本，如果用 v1.0.1 还会遗留一个 cve 漏洞

Comment (Jason-ZW) [reply go.mod]: 1.0.2 只比 1.0.1 多了一个[ git reset 的修复](https://github.com/rancher/wrangler/commit/a7c0046a4e335f44d0be423aed4499ad5103e7ab)，从cve描述来看1.0.0以上版本可以已经得到修复也就是说1.0.1已经修复了对应的wrangler cve，你说的1.0.1还会遗留一个漏洞是哪个漏洞呢？

Comment (smallteeths) [reply go.mod]: 1.0.1 昨天测试有以下 cve 漏洞 （这里我理解错了我看我们 rancher 已经更新到了 wrangler 1.1.0 版本，确实 gke 其他的 operator 依赖更新的都比较慢）

| image                        | rancher_release            | package_name                  | type                      | vulnerability_id  | severity | url                                       | target              | patched_version                              | mirrored | state           | notes |
|------------------------------|---------------------------|-------------------------------|---------------------------|-------------------|----------|-------------------------------------------|---------------------|----------------------------------------------|----------|----------------|-------|
| wangsiye/ack-operator:v2.7.0-update.49 |                           | github.com/rancher/wrangler | suse linux enterprise server | CVE-2022-31249 | HIGH     | https://avd.aquasec.com/nvd/cve-2022-31249 | usr/bin/ack-operator | "0.7.4-security1, 0.8.5-security1, 0.8.11, 1.0.1" | False    | pending review |       |

Comment (smallteeths) [reply go.mod]: done

**Review by Jason-ZW — APPROVED** [review]

**Pull request merged**